sdk/iot/Azure.Iot.Hub.Service/src/StatisticsClient.cs

```diff
@@ -1,25 +1,76 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using System;
-using System.Collections.Generic;
-using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Iot.Hub.Service.Models;
 
 namespace Azure.Iot.Hub.Service
 {
     /// <summary>
-    /// Statistics Client place holder
+    /// Statistics client to acquire information about IoT Hub statistics.
     /// </summary>
     public class StatisticsClient
     {
+        private readonly StatisticsRestClient _statisticsRestClient = null;
+
+        /// <summary>
+        /// Initializes a new instance of StatisticsClient.
+        /// </summary>
+        protected StatisticsClient()
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of StatisticsClient.
+        /// <param name="statisticsRestClient"> The REST client to query statistics of the IoT Hub. </param>
+        /// </summary>
+        internal StatisticsClient(StatisticsRestClient statisticsRestClient)
+        {
+            Argument.AssertNotNull(statisticsRestClient, nameof(statisticsRestClient));
+
+            _statisticsRestClient = statisticsRestClient;
+        }
+
+        /// <summary>
+        /// Gets devices statistics of the IoT Hub identity registry.
+        /// </summary>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        /// <returns>The device statistics of the IoT Hub.</returns>
+        public virtual Response<DevicesStatistics> GetDeviceStatistics(CancellationToken cancellationToken = default)
```

Comment:
Small nitpick, but if the method is called "GetDeviceStatistics" then shouldn't the return type be "DeviceStatistics" instead of "Device**s**Statistics"?

Reply:
This issue is present in the comments around here, too. Consistency would be helpful

Reply:
That's a fair point, will make the change.